grails-app/conf/spring/resources.groovy

```diff
@@ -4,7 +4,11 @@ import au.org.ala.collectory.ExtendedPluginAwareResourceBundleMessageSource
 beans = {
     // Custom message source
     messageSource(ExtendedPluginAwareResourceBundleMessageSource) {
-        basenames = ["WEB-INF/grails-app/i18n/messages"] as String[]
+        basenames = [
+                "file:///var/opt/atlas/i18n/collectory-plugin/messages",
+                "file:///opt/atlas/i18n/collectory-plugin/messages",
+                "WEB-INF/grails-app/i18n/messages"
```

Comment:
Just checking if Spring loads all available bundles and then picks the first/last (order is significant) or whether it stops after it finds "one" bundle and just uses that. Quick Googling suggests the latter but would be good to confirm. Did you test this with 2 i18n codes - one in WEB-INF and another external to see which code "won"?

Reply:
Found answer [here](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/AbstractResourceBasedMessageSource.html#setBasenames-java.lang.String...-): 

> The associated resource bundles will be checked sequentially when resolving a message code. Note that message definitions in a previous resource bundle will override ones in a later bundle, due to the sequential lookup.

Reply:
Yes, I've tested modifying the main messages in the three directories with numbers (1, 2, 3) to see which one gets precedence. Also removing the customization to check that does works as expected. 

Anyway was not so straightforward in other similar PRs:
https://github.com/search?q=is%3Apr+author%3Avjrj+org%3AAtlasOfLivingAustralia+ala-i18n&type=Issues

Thanks @nickdos .